themes/90210/README.md

```diff
@@ -0,0 +1,15 @@
+# 90210 💜
+
+![alt text](90210-dark.png)
```

Comment:
Could you use meaningful texts instead of `alt text`. The text "`alt text`" is actually the explanation of Markdown grammar.